axelrod/tests/strategies/test_worse_and_worse.py

```diff
@@ -1,11 +1,10 @@
 """Tests for the WorseAndWorse strategies."""
 
 import axelrod
-from .test_player import TestPlayer
+from test_player import TestPlayer
```

Comment:
The tests are currently failing because of this change. You need to change it back to:

```
from .test_player import TestPlayer
```

If you wanted to run the tests locally you can do this by typing the following at the command line from the root of the Axelrod directory:

```
$ python -m unittest axelrod.tests.unit.test_worse_and_worse
```

More information about that here: http://axelrod.readthedocs.io/en/latest/tutorials/contributing/strategy/running_tests.html